auth-foundation/src/test/java/com/okta/authfoundation/credential/CredentialTest.kt

```diff
@@ -126,6 +144,15 @@ class CredentialTest {
         assertThat(removedEvent.credential).isEqualTo(credential)
     }
 
+    @Test fun testGetTokenFlowEmitsSingleNullAfterRemove(): Unit = runBlocking {
+        val storage = mock<TokenStorage>()
+        val credentialDataSource = mock<CredentialDataSource>()
+        val token = createToken()
+        val credential = oktaRule.createCredential(token = token, tokenStorage = storage, credentialDataSource = credentialDataSource)
+        credential.delete()
+        assertThat(credential.getTokenFlow().single()).isNull()
```

Comment:
This case seems like it should instantly be closed, and not return any items.

Reply:
k, I do think it should emit a final `null` value for anyone already observing a token flow.

I'll see what I can come up with to return an empty flow if the credential was already deleted without changing the final `null` behavior